src/locale/locales/it/messages.po

```diff
@@ -2213,7 +2213,7 @@ msgstr "I feed sono algoritmi personalizzati che gli utenti creano con un minimo
 
 #: src/screens/Onboarding/StepTopicalFeeds.tsx:80
 #~ msgid "Feeds can be topical as well!"
-#~ msgstr "I feeds possono anche avere tematiche!"
+#~ msgstr "I feed possono anche avere tematiche!"
```

Comment:
Also just as a tip for the future, you don't need to bother changing any of the lines that start with `#~` as these are obsolete messages that have been removed from the codebase. So you can just ignore them or you can delete these lines if you prefer :)